tests/manual/monkey/README.md

```diff
@@ -7,29 +7,27 @@ This is a testing tool developed to support manual ad-hoc testing of individual
 
 ## Prerequisites
 
-JavaFX SDK is required to build the tool.  You can use a JavaFX SDK that you build or you can download the JavaFX SDK.
-The latest SDK can be found here:
+JavaFX SDK 24+ and JDK 23+ are required to build and run the tool.
+You can use a JavaFX SDK that you build or you can download the JavaFX SDK found here:
 
-https://jdk.java.net/javafx21/
+https://jdk.java.net/
 
 
 ## Build
 
-The tool uses `ant` to build a non-modular JAR.  You'll need to specify the path to JavaFX SDK 20+
-(using absolute path, the script apparently does not understand ~ symbols):
+The tool uses `ant` to build a non-modular JAR.  You'll need to specify the path to JavaFX SDK
+using absolute path:
 ```
 ant -Djavafx.home=<JAVAFX>
 ```
 
 
 ## Run
 
-The tool requires JDK 21+ and JavaFX 21+.
-
 To launch, specify the path to the JavaFX SDK lib/ folder on the command line, example:
 
 ```
-java -p <JAVAFX>/javafx-sdk-21/lib/ --add-modules ALL-MODULE-PATH -jar MonkeyTester.jar
+java -p <JAVAFX>/javafx-sdk-24/lib/ --add-modules ALL-MODULE-PATH -jar MonkeyTester.jar
```

Comment:
I used the README to build and run this app.
Suggestion for future updates:
In this command we should not add "javafx-sdk-24" in the path.
JAVAFX already includes sdk folder we just need to use `java -p <JAVAFX>/javafx-sdk-24/lib/ --add-modules ALL-MODULE-PATH -jar MonkeyTester.jar`

Reply:
good point, thanks!

Created https://bugs.openjdk.org/browse/JDK-8353743